pages/common/xdg-user-dirs-update.md

```diff
@@ -0,0 +1,12 @@
+# xdg-user-dirs-update
+
+> Update XDG user directories.
+> More information: <https://manpages.ubuntu.com/manpages/bionic/man1/xdg-user-dirs-update.1.html>.
+
+- Change XDG's DESKTOP directory to the specified directory (must be absolute):
+
+`xdg-user-dirs-update --set DESKTOP "{{/path/to/directory}}"`
```

Comment:
Please remove the preceding slash from all path tokens.

Reply:
Fixed in #3711. This flew under the radar.